### ISSUE TITLE
flavors returns 2 identical flavors with id "1"

### ISSUE THREAD

**Issue by bodenr** [softlayer/jumpgate#25]
If you setup jumpgate + novaclient and then do a `nova flavor-list` you get the following flavors back:

```
+----+------------------------+-----------+------+-----------+------+-------+-------------+-----------+
| ID | Name                   | Memory_MB | Disk | Ephemeral | Swap | VCPUs | RXTX_Factor | Is_Public |
+----+------------------------+-----------+------+-----------+------+-------+-------------+-----------+
| 1  | 1 vCPU, 1GB ram, 25GB  | 1024      | 25   | 0         |      | 1     | 1           | True      |
| 2  | 1 vCPU, 1GB ram, 100GB | 1024      | 100  | 0         |      | 1     | 1           | True      |
| 3  | 2 vCPU, 2GB ram, 100GB | 2048      | 100  | 0         |      | 2     | 1           | True      |
| 4  | 4 vCPU, 4GB ram, 100GB | 4096      | 100  | 0         |      | 4     | 1           | True      |
| 5  | 8 vCPU, 8GB ram, 100GB | 8192      | 100  | 0         |      | 8     | 1           | True      |
| 1  | 1 vCPU, 1GB ram, 25GB  | 1024      | 25   | 0         |      | 1     | 1           | True      |
+----+------------------------+-----------+------+-----------+------+-------+-------------+-----------+
```

Notice how there are 2 identical flavors with id `1`.

I didn't poke at this, but likely due to [FLAVORS[None] = FLAVORS[1]](https://github.com/softlayer/jumpgate/blob/master/jumpgate/compute/drivers/sl/flavors.py#L41)

**Comment by sudorandom** [comment]
Thanks for the detail. This is an extremely easy fix.

**Comment by beittenc** [comment]
Closed by #26
